scripts/lint.sh

```diff
@@ -73,6 +73,15 @@ function test_require_equal_zero {
   fi
 }
 
+function test_require_len_zero {
+  if grep -R -o -P 'require\.Len\((t, )?.+?, 0(,|\))' .; then
```

Comment:
```suggestion
  if grep -R -o -P 'require\.Len\((t, )?.+, 0(,|\))' .; then
```
Why do we need this `?` here? and if we do need it... I feel like `.*` is better than `.+?`... Unless I'm totally off on my regex

Reply:
we don't, extra from an earlier version of this regex